ciao-controller/main.go

```diff
@@ -89,8 +89,8 @@ func main() {
 
 	context.BlockDriver = func() storage.BlockDriver {
 		driver := storage.CephDriver{
-			SecretPath: *keyringPath,
-			ID:         *cephID,
+			SecretPath: keyringPath,
```

Comment:
Aren't we initialising this too early?  As far as I can tell, keyringPath and cephID will still contain their defaults at this point.  Changing the global variables after this point isn't going to do anything.